File: startingPoint/02-EstruturaSequencial/Exercicios_Propostos/06.py
#!/usr/bin/python
# coding: latin-1

# 06 - Faça um programa que receba o salário base de um funcionário, calcule e mostre o seu salário a receber, sabendo-se que o funcionário
# tem gratificação de R$50 e paga imposto de 10% sobre o salário base

sal_base = float(input("Digite o valor do seu salário aqui: "))

grat = 50
imp = sal_base * 10 / 100

sal_receber = (sal_base + grat - imp)

print("O salário a receber será: R$", sal_receber)

File: SRC - PERL/00 - Estruturas Sequenciais/06.pl
#!/usr/bin/env perl
use v5.10;

use strict;
use warnings;

use utf8;
use open qw(:std :utf8);

# Faça um programa que receba o salário base de um funcionário, calcule e mostre o seu salário a receber, sabendo-se que o funcionário tem gratificação de R$50 e paga imposto de 10% sobre o salário base

print "Digite o salário base > ";
my $salario_base = <>;
chomp $salario_base;

my $imposto = $salario_base * 10 / 100;

my $novo_salario = $salario_base + 50 - $imposto;

printf "O novo salário é R\$ %.2f", $novo_salario;
say "";
